patchy/api.py

```diff
@@ -148,25 +149,57 @@ def _get_source(func):
         return source
 
 
-def _set_source(func, new_source):
+def _class_name(func):
```

Comment:
`func.im_class.__name__` but it was replaced with `__qualname__` because `__qualname__` is 'easier'